### ISSUE TITLE
Is it possible to pass several parameters in archiveRecoverySettings during PITR init ?

### ISSUE THREAD

**Issue by davidberardozzi** [sorintlab/stolon#297]
Hi,

And thank you a lot @sgotti and all other contributors for this master piece of work !
We recently tested recovery feature with wal-e in production environment. It worked like a charm.

But I first fell into this issue:
* When creating last base backup we were in timeline 2
* When trying to restore, everything went well but last timeline was not restored (timeline 4 in our case)
* I had to add `recovery_target_timeline =  latest` manually in recovery.conf

I initially tried to add it to the PITR command like this:

```
stolonctl --cluster-name=gup-cluster --store-backend=etcd --store-endpoints=etcd:2379 init '{ "initMode": "pitr", "pitrConfig": { "dataRestoreCommand": "envdir /etc/wal-e.d/env wal-e backup-fetch %d LATEST" , "archiveRecoverySettings": { "restoreCommand": "envdir /etc/wal-e.d/env wal-e wal-fetch \"%f\" \"%p\"", "recovery_target_timeline": "latest"} } }'
```

But this had no effect on recovery.conf.

Am I doing things wrong, can I add several parameters to recovery_target_timeline ?

Thank you

**Comment by sgotti** [comment]
@davidberardozzi It'll be possible to add additional recovery.conf parameters but not yet implemented: https://github.com/sorintlab/stolon/blob/master/pkg/cluster/cluster.go#L159

recovery.conf is wrote by the keeper so it'll be overwritten if changed manually.

I can work on this in 2 weeks when I'll be back from some vacation (if someone isn't faster than me creating a PR).

**Comment by davidberardozzi** [comment]
Ok thanks

**Comment by sgotti** [comment]
@davidberardozzi created an initial pull request #303 

You can set recovery target settings in this way:

```
stolonctl --cluster-name=gup-cluster --store-backend=etcd --store-endpoints=etcd:2379 init '{ "initMode": "pitr", "pitrConfig": { "dataRestoreCommand": "envdir /etc/wal-e.d/env wal-e backup-fetch %d LATEST" , "archiveRecoverySettings": { "restoreCommand": "envdir /etc/wal-e.d/env wal-e wal-fetch \"%f\" \"%p\"" }, "recoveryTargetSettings": { "recoveryTargetTimeline": "latest"} } }'
```

if you can test it and report if it works I'll be very happy.

**Comment by davidberardozzi** [comment]
@sgotti, Thanks a lot. Sorry I won't be able to test this whithin next week at least.

**Comment by sgotti** [comment]
@davidberardozzi merged #303. I'm closing this. If you manage to test it and find some issues please let us know. Thanks!
